File: src/python/tutorias/domain/usuario.py
from abc import ABC


class Usuario(ABC):
    def __init__(self, id_usuario: int, nombre: str, correo: str):
        self._id = id_usuario
        self._nombre = nombre
        self._correo = correo

    def obtener_perfil(self) -> str:
        return f"{self._nombre} - {self._correo}"

File: src/java/edu/uees/tutorias/domain/Usuario.java
package edu.uees.tutorias.domain;

public abstract class Usuario {

    protected int id;
    protected String nombre;
    protected String correo;

    public Usuario(int id, String nombre, String correo) {
        this.id = id;
        this.nombre = nombre;
        this.correo = correo;
    }

    public String obtenerPerfil() {
        return nombre + " - " + correo;
    }

    public String getCorreo() {
        return correo;
    }
}